### ISSUE TITLE
Plot values from csv/tsv file columns

### ISSUE THREAD

**Issue by mechatroner** [Henoc/vscode-plot#2]
Hello!
It would be cool to plot histograms for specific columns in csv files or 2-D plots for 2 specific columns in csv file. You can use CSV-parsing functions from [Rainbow CSV](https://github.com/mechatroner/vscode_rainbow_csv) plugin. What do you think about this?

**Comment by ahundt** [comment]
I agree! it would be great if this could work for columns in addition to rows
